exporter/prometheus/prometheus.go

```diff
@@ -196,6 +196,12 @@ func (c *collector) Describe(ch chan<- *prometheus.Desc) {
 // and delivers them as Prometheus Metrics.
 // Collect is invoked everytime a prometheus.Gatherer is run
 // for example when the HTTP endpoint is invoked by Prometheus.
+// Each OpenCensus AggregationData will be converted to
```

Comment:
This new comments section is not readable by the user given collector is not exported, we can add it to the Export's godoc.

Reply:
Yeah just realized that - I've moved it to ExportView().